ramls/circulation.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Circulation
-version: v7.4
+version: v7.5
```

Comment:
This change also needs removing, based upon our conversation about the interface version not changing

Reply:
done